app/User/ReverseProxyUserProvider.php

```diff
@@ -133,7 +135,11 @@ public function getUsername()
      */
     public function getName()
     {
-        return '';
+        if (! REVERSE_PROXY_NAME_HEADER ){
+            return '';
+	}else{
+            return $this->name;
+	}
```

Comment:
Do you mind indenting your code properly? It must be consistent with the rest of the code.

Reply:
tab replace by space (need to find tips about code indent in terminal)

Reply:
the `else` have to be surrounded by 1 space (for all your changes). Look at the GitHub diff.